packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -732,10 +749,17 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         return this.channel.applyStashedOp(innerContents.content);
     }
 
-    private verifyNotClosed() {
+    private verifyNotClosed(methodName: string, checkTombstone = true) {
```

Comment:
You should change this param from "methodName" to "errorMessage" and then pass in any string. It's more of a message that should be part of the error.

Reply:
The message needs to be different if tombstone is on or not, I've opted not to change the method name as that would need to be the same once tombstone is removed.

Reply:
Please review my change

Reply:
Yes, this is what I meant, thanks!